api/pkg/app/gorm_logger.go

```diff
@@ -69,7 +69,7 @@ func (w *prodWriter) Printf(format string, data ...interface{}) {
 				log = log.With("unknown", d)
 			}
 		default:
-			log = log.With("unknown", d)
+			_ = dt
```

Comment:
`log = log.With("unknown", d)` any reason this was removed?

Reply:
Will add it

Reply:
Removed by mistake while resolving the lint error